src/Pages/DashBoard.tsx

```diff
@@ -0,0 +1,166 @@
+              </Heading>
+            </Box>
+            <Box textAlign="left" m="1" maxW="800px" minW="300px" p="1.5rem">
+              {userAns &&
+                quizData.data[quizid].questions.map((question) => {
+                  const ansObj = userAns.find(
+                    (ans) => ans.questionId === question.questionId
+                  );
+                  return (
+                    <Box className="glass-effect-2" p="0.5rem" mb="1rem">
+                      <Text
+                        fontSize="xl"
+                        p="0.5rem"
+                        fontWeight="bold"
+                        mb="2"
+                        mt="2"
+                      >
+                        {question.questionId}.{question.question}
+                      </Text>
+                      {question.options.map((option, idx) => {
+                        return (
+                          <Text mb="2" ml="4" fontSize="lg" key={idx}>
+                            {option.text}
+                            <span>
+                              {ansObj && ansObj.answer === option.optionId ? (
+                                ansObj.answer === question.answer ? (
+                                  <CheckIcon
+                                    bg="#FF9478"
+                                    p="5px"
+                                    borderRadius="50%"
+                                    ml="1rem"
+                                    boxSize={6}
+                                    color="white"
+                                  />
+                                ) : (
+                                  <CloseIcon
+                                    bg="#FF9478"
+                                    p="5px"
+                                    ml="1rem"
+                                    borderRadius="50%"
+                                    boxSize={6}
+                                    color="white"
+                                  />
+                                )
+                              ) : option.optionId === question.answer ? (
+                                <CheckIcon
+                                  bg="#FF9478"
+                                  p="5px"
+                                  borderRadius="50%"
+                                  ml="1rem"
+                                  boxSize={6}
+                                  color="white"
+                                />
+                              ) : (
+                                <span />
+                              )}
+                            </span>
+                          </Text>
+                        );
+                      })}
+                      
+                    </Box>
+                  );
+                })}
+              <Box>
+                <Box>
+                  {!userAns && (
+                    <Box>
+                      {" "}
```

Comment:
You missed this space :)
Also, avoid adding extra `Box` elements. Use `<> </>` instead. `Box` adds extra divs to the DOMTree